src/config/section/compute.js

```diff
@@ -104,7 +104,8 @@ export default {
           icon: 'reload',
           label: 'label.action.reboot.instance',
           dataView: true,
-          show: (record) => { return ['Running'].includes(record.state) }
+          show: (record) => { return ['Running'].includes(record.state) },
+          args: ['bootintosetup']
```

Comment:
```suggestion
          args: (record) => { return record.hypervisor === 'VMware' ? ['bootintosetup'] : [] }
```
@DaanHoogland 
This should work but currently it seems record is coming undefined from AutogenView. I've created PR #327 to fix that.
`record` here will be the response for `listvirtualmachinesmetrics` response item

Reply:
ah to bad, I want to know the hypervisor type for the machine. @shwstppr any tips?

Reply:
@DaanHoogland `record.hypervisor` with give you that.

VMs in primate are listed with listvirtualmachinesmetrics API and it only has hypervisor param. No parameter named hypervisortype in API response.

Reply:
hypervisor contains the type, tnx. There is something confusing in the API doc, it only lists it as a request parameter. I think I should be fine.